assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss

```diff
@@ -22,4 +22,212 @@
+    @include mq($from: phablet) {
+      display: inline-block;
+      width: 50%;
+      vertical-align: top;
+    }
+
+    @include mq($from: desktop) {
+      width: 485px;
+    }
+
+    @include mq($from: leftCol) {
+      width: 520px;
+    }
+  }
+
+  .product-block__image {
+    border-bottom: 1px solid gu-colour(garnett-neutral-4);
+    padding: 0 ($gu-h-spacing / 2);
+    margin-bottom: $gu-v-spacing / 2;
+
+    @include mq($from: desktop) {
+      position: relative;
+    }
+
+    .svg-penny-farthing-circles {
+      display: none;
+
+      @include mq($from: desktop) {
+        display: block;
+        position: absolute;
+        width: 250px;
+        bottom: 0;
+        left: 255px;
+      }
+
+      @include mq($from: leftCol) {
+        left: 220px;        
+      }
+    }
+  }
+
+  .product-block__heading {
+    font-family: $gu-egyptian-web;
+    font-weight: 200;
+    font-size: 22px;
+    line-height: 26px;
+    padding: 0 ($gu-h-spacing / 2) $gu-v-spacing;
+
+    @include mq($from: phablet) {
+      font-size: 28px;
+      line-height: 1.14;
+      width: 700px;
+    }
+
+    @include mq($from: desktop) {
+      font-size: 36px;
+      line-height: 1.11;
+      width: 600px;
+      padding-bottom: $gu-v-spacing * 3;
+    }
+  }
+
+  .product-block__copy {
+    padding: 0 ($gu-h-spacing / 2) $gu-v-spacing;
+  }
+
+  .product-block__product-heading {
+    color: #fff;
+    font-weight: bold;
+    font-family: $gu-egyptian-web;
+    line-height: 1;
+    display: inline;
+    padding: 0 5px 3px;
+  }
+
+  .product-block__product-description {
+    margin: $gu-v-spacing / 2 0 $gu-v-spacing;
+    font-size: 16px;
+    line-height: 1.25;
+    font-family: $gu-egyptian-web;
+  }
+
+  .component-feature-list {
+    font-size: 16px;
+    line-height: 1.25;
+    font-family: $gu-egyptian-web;
+    list-style-image: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 10 10"><circle cx="5" cy="5" r="5" fill="#ff4e36"/></svg>');
+    margin-left: 26px;
+
+    @include mq($from: leftCol) {
+      columns: 2;
+      column-gap: 54px;
+    }
+  }
+
+  .component-feature-list__heading {
+    font-weight: bold;
+    display: inline;
+    vertical-align: top;
+  }
+
+  .component-feature-list__item {
+    margin-bottom: $gu-v-spacing / 2;
+  }
+
+  .product-block__highlight {
+    font-weight: normal;
+    background-color: gu-colour(news-garnett-highlight);
+    padding: 0 2px;
+  }
+
+  .product-block__product--premium-app {
+    .product-block__product-heading {
+      background-color: gu-colour(news-garnett-media-main-1);
+    }
+  }
+
+  .product-block__product--daily-edition {
+    .component-grid-image {
+      @include mq($from: desktop) {
+        position: relative;
+        left: -100px;
+      }
+
+      @include mq($from: leftCol) {
+        left: -135px;
+      }
+    }
+
+    .product-block__product-heading {
+      background-color: gu-colour(sport-garnett-media-main-1);
```

Comment:
Maybe we should introduce a new colour here? If the sport colour changes we wouldn't necessarily want this to change along with it.

Reply:
As discussed offline, we're going to look into giving the palette more support-related names.